### PR TITLE
Updating functions of BlockPoolUI 

### PR DESCRIPTION
**Pull request by shivamdurgbuns**
Updating UI functions of BlockPoolUI so that it supports ODF 4.13  
No changes are done since OCS 4.8

### CODE REVIEW

File: ocs_ci/ocs/ui/block_pool.py [hunk 1]
```diff
@@ -64,7 +64,7 @@ def check_pool_existence(self, pool_name):
             bool: True if pool is in the list of pools page, otherwise False
 
         """
-        self.navigate_cluster_overview_page()
+        # self.navigate_cluster_overview_page()
```
Comment (ramkiperiy) [inline ocs_ci/ocs/ui/block_pool.py:67]: why not remove this line?

Comment (shivamdurgbuns) [reply ocs_ci/ocs/ui/block_pool.py]: removed all in the lastest commit, I forgot to push that

File: ocs_ci/ocs/ui/base_ui.py [hunk 2]
```diff
@@ -882,8 +883,18 @@ def navigate_block_pool_page(self):
 
         """
         logger.info("Navigate to block pools page")
-        self.navigate_to_ocs_operator_page()
-        self.do_click(locator=self.page_nav["block_pool_link"])
+        self.nav_odf_default_page()
```
Comment (DanielOsypenko) [inline ocs_ci/ocs/ui/base_ui.py:886]: please use this approach, so we do not repeat code and save time on maintenance. Each page prints it's name when navigating into it

            PageNavigator()
            .nav_odf_default_page()
            .nav_storage_systems_tab()
            .nav_storagecluster_storagesystem_details()
            .nav_ceph_blockpool()

File: ocs_ci/ocs/ui/views.py [hunk 3]
```diff
@@ -228,6 +228,11 @@
     "check_first_row_checkbox": ('input[name="checkrow0"]', By.CSS_SELECTOR),
     "remove_search_filter": ('button[aria-label="close"]', By.CSS_SELECTOR),
     "delete_resource_kebab_button": ('//*[contains(text(), "Delete")]', By.XPATH),
+    "ocs-storagecluster-storagesystem": (
+        '//a[text()="ocs-storagecluster-storagesystem"]',
+        By.XPATH,
+    ),
+    "block_pool_tab": ('//span[normalize-space()="BlockPools"]', By.XPATH),
```
Comment (DanielOsypenko) [inline ocs_ci/ocs/ui/views.py:235]: we already have this locator
views.py -> validation_4_11. 
It should be available from self.validation_loc["blockpools"]

File: ocs_ci/ocs/ui/views.py [hunk 4]
```diff
@@ -228,6 +228,11 @@
     "check_first_row_checkbox": ('input[name="checkrow0"]', By.CSS_SELECTOR),
     "remove_search_filter": ('button[aria-label="close"]', By.CSS_SELECTOR),
     "delete_resource_kebab_button": ('//*[contains(text(), "Delete")]', By.XPATH),
+    "ocs-storagecluster-storagesystem": (
+        '//a[text()="ocs-storagecluster-storagesystem"]',
```
Comment (DanielOsypenko) [inline ocs_ci/ocs/ui/views.py:232]: please check this locator from views validation_4_9["ocs-storagecluster-storgesystem"] it should be the same

File: ocs_ci/ocs/ui/views.py [hunk 5]
```diff
@@ -822,6 +827,28 @@
     "confirm_add_capacity": ('button[data-test-id="confirm-action"]', By.CSS_SELECTOR),
 }
 
+block_pool_4_12 = {
```
Comment (DanielOsypenko) [inline ocs_ci/ocs/ui/views.py:830]: if block_pool_4_12 and block_pool_4_13 we don't need to repeat them. The point of this dict is to reuse locators. Just pass "block_pool" : block_pool_4_12 for locators["4.13"]

Comment (shivamdurgbuns) [reply ocs_ci/ocs/ui/views.py:825]: Made the changes.

File: ocs_ci/ocs/ui/views.py (hunk 4 above)
Comment (shivamdurgbuns) [reply ocs_ci/ocs/ui/views.py]: Made the changes.

File: ocs_ci/ocs/ui/views.py (hunk 3 above)
Comment (shivamdurgbuns) [reply ocs_ci/ocs/ui/views.py]: Made the changes.

File: ocs_ci/ocs/ui/base_ui.py (hunk 2 above)
Comment (shivamdurgbuns) [reply ocs_ci/ocs/ui/base_ui.py]: Made the changes.

File: ocs_ci/ocs/ui/base_ui.py [hunk 6]
```diff
@@ -882,8 +883,14 @@ def navigate_block_pool_page(self):
 
         """
         logger.info("Navigate to block pools page")
-        self.navigate_to_ocs_operator_page()
-        self.do_click(locator=self.page_nav["block_pool_link"])
+        blockpool_page = (
+            PageNavigator()
+            .nav_odf_default_page()
+            .nav_storage_systems_tab()
+            .nav_storagecluster_storagesystem_details()
```
Comment (DanielOsypenko) [inline ocs_ci/ocs/ui/base_ui.py:890]:
```
PageNavigator()
.nav_odf_default_page()
.nav_storage_systems_tab()
.nav_storagecluster_storagesystem_details()
``` returns storage_system_details page, not blockpool_page this way you will invoke nav_ceph_blockpool from storage_system_details which is logical -> `blockpool_page.nav_ceph_blockpool()`

File: ocs_ci/ocs/ui/views.py [hunk 7]
```diff
@@ -822,6 +822,20 @@
...
+    "actions_inside_pool": (
+        "//span[text()='Actions']/..",
+        By.XPATH,
+    ),
+    "delete_pool_inside_pool": (
```
Comment (DanielOsypenko) [inline ocs_ci/ocs/ui/views.py:830]: we have same loc in line 458

**Review by DanielOsypenko — APPROVED** [review]

**Comment by DanielOsypenko** [comment]
@shivamdurgbuns looks great! Do you want to backport it to older versions?

**Review by ramkiperiy — APPROVED** [review]

**Pull request merged**